dashboard/app/models/concerns/curriculum/course_types.rb

```diff
@@ -29,22 +29,16 @@ def must_have_same_course_type_as_family
     errors.add(:instruction_type, 'must be the same for all courses in a family.') if all_family_courses.map(&:instruction_type).any? {|type| type != instruction_type}
   end
 
-  # Get the family name for the course based on if its set on the UnitGroup or Unit
-  def get_course_family_name
-    is_a?(Script) && unit_group ? unit_group.family_name : family_name
-  end
-
   # If course we are check is a unit_group or a unit that is in a unit_group check the family_name on the UnitGroup.
   # If the course is a unit that is not in a unit_group check the unit for the family_name
   def get_family_courses
-    family_name = get_course_family_name
     return nil if family_name.nil_or_empty?
 
     all_family_courses = nil
 
-    if is_a?(UnitGroup) || (is_a?(Script) && unit_group)
+    if is_a?(UnitGroup)
```

Comment:
According to the comment, if this is a script in a unit group we should be checking the unit group's family name right? I think that's what you're removing here, but should this method return `nil` or the empty array?

Reply:
Updated the comment